### PR TITLE
Feature/#3357/admin cabinet clients orders page sorting

### PR DESCRIPTION
**Pull request by TarasKuzma1999**

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-orders/ubs-admin-customer-orders.component.ts [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    });
+  }
+
+  @HostListener('window:resize', ['$event'])
+  onResize() {
+    this.setTableResize(this.matTableRef.nativeElement.clientWidth);
+  }
+
+  ngOnDestroy() {
+    this.destroy$.unsubscribe();
```
Comment (NewArtMask) [inline src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-orders/ubs-admin-customer-orders.component.ts:230]: this.destroy$.next() ?

File: src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-orders/ubs-admin-customer-orders.component.ts [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    });
+  }
+
+  @HostListener('window:resize', ['$event'])
+  onResize() {
+    this.setTableResize(this.matTableRef.nativeElement.clientWidth);
+  }
+
+  ngOnDestroy() {
+    this.destroy.next();
```
Comment (NewArtMask) [inline src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-orders/ubs-admin-customer-orders.component.ts:230]: $ :-)

**Review by NewArtMask — APPROVED** [review]

**Review by KhrystynaPr — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customers.component.html [hunk 3]
```diff
@@ -171,8 +171,14 @@
...
-          [class.pointer]="column.title.key === 'clientName'"
-          (click)="column.title.key === 'clientName' && openCustomer(row, row[column.title.key])"
+          [class.pointer]="column.title.key === 'clientName' || column.title.key === 'number_of_orders'"
+          (click)="
+            column.title.key === 'clientName'
+              ? openCustomer(row, row[column.title.key])
+              : column.title.key === 'number_of_orders'
+              ? openOrders(row)
+              : null
+          "
```
Comment (hnativlyubomyr) [inline src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customers.component.html:181]: please, create click handler here

File: src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-orders/ubs-admin-customer-orders.component.ts [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+    });
+  }
+
+  private checkResizing(event: any, index: any) {
+    const cellData = this.getCellData(index);
+    if (index === 0 || (Math.abs(event.pageX - cellData.right) < cellData.width / 2 && index !== this.columns.length - 1)) {
+      this.isResizingRight = true;
+    } else {
+      this.isResizingRight = false;
+    }
```
Comment (hnativlyubomyr) [inline src/app/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-orders/ubs-admin-customer-orders.component.ts:173]: remove if..else. Set condition value for isResizingRight

**Review by s1auka — APPROVED** [review]

**Review by slavafisher — APPROVED** [review]

**Review by Yura935 — APPROVED** [review]

**Pull request merged**